app/views/layouts/_footer.html.erb

```diff
@@ -31,7 +30,6 @@
             <% end %>
           </div>
           <div class="col-6 footer-links">
-            <h6>Info</h6>
```

Comment:
Unrelated

Reply:
sorry about this. 
This is the previous commit. somehow it got attached. I'll fix it.

Reply:
@tachyons I made changes to the form and this time it's not a workaround. When I'm making a pull request the above commits I made previously are also adding up to the current one. Can you please help me.